Qualysv2 new args

iulian509 opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added optional arguments to qualys-schedule-scan-create command.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

dorschw commented:
Hi @iulian509, thank you for the contribution!
Due to the local holiday season, reviewing your PR may be slightly delayed. We'll review it as soon as we can.

esharf commented:
@iulian509 looks good.
do you want to schedule a short demo to ensure the command is working correctly?
if so let me know in [DFIR](https://dfircommunity.slack.com/team/UQWL4FQJ0).
Anyway, I will make some validations and push it forward.
Thanks a lot for your contribution.

iulian509 commented:
@esharf I Just realized the previous commit was wrong. I fixed qualys-schedule-scan-create command and added the same new arguments to qualys-schedule-scan-update command. I have also removed the required tag on option_title argument because only one is needed (option_id or option_title), but you can't use both (as you can see on the page 55 https://www.qualys.com/docs/qualys-api-vmpc-user-guide.pdf).

On the other hand I will joint the slack community so we can schedule a demo.

Sorry for the inconvenience.
